iceoryx_binding_c/include/iceoryx_binding_c/event_info.h

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) 2020 by Apex.AI Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef IOX_BINDING_C_TRIGGER_INFO_H
+#define IOX_BINDING_C_TRIGGER_INFO_H
+
+#include "iceoryx_binding_c/internal/c2cpp_binding.h"
+#include "iceoryx_binding_c/subscriber.h"
+#include "iceoryx_binding_c/user_trigger.h"
+
+/// @brief event info handle
+typedef const CLASS EventInfo* iox_event_info_t;
+
+/// @brief returns the id of the event
+/// @param[in] self handle to event info
+/// @return eventId
+uint64_t iox_event_info_get_event_id(iox_event_info_t const self);
+
+/// @brief does the event originate from a certain subscriber
+/// @param[in] self handle to event info
+/// @param[in] subscriber handle to the subscriber in question
+/// @return true if the event originates from the subscriber, otherwise false
+bool iox_event_info_does_originate_from_subscriber(iox_event_info_t const self, iox_sub_t const subscriber);
```

Comment:
minor nitpicking. In order to make the function names smaller this could be something like `iox_event_info_origin_is_subscriber`

Reply:
nitpicking rejected ;)